Exception Handling/Task1/Program.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+
+namespace Task1
+{
+    internal class Program
+    {
+        private static void Main(string[] args)
+        {
+            try
+            {
+                PrintFirstCharacter();
+            }
+            catch (ArgumentNullException e)
```

Comment:
You have same catch blocks for two exceptions, so you can siplify this by using only one block
https://stackoverflow.com/questions/136035/catch-multiple-exceptions-at-once